fastembed/text/colpali_model.py

```diff
@@ -0,0 +1,88 @@
+from typing import Any, Dict, Iterable, List
+
+import numpy as np
+
+from fastembed.common.onnx_model import OnnxOutputContext
+from fastembed.text.onnx_embedding import OnnxTextEmbedding
+
+supported_onnx_models = [
+    {
+        "model": "akshayballal/colpali-v1.2-merged",
```

Comment:
It's not really separated in our quantization. We could re-quantize it and separate, but I can not say it will have original behaviour. It could, but I'm not sure 100%

Reply:
Which image model is used? 
How much does it weight? 
Does it make sense to split the models or the difference is negligible?

Reply:
Gemma 2B